### ISSUE TITLE
can't get libgranite ! also OS system can't be updated entirely

### ISSUE THREAD

**Issue by AkoZ** [elementary/granite#620]
### What Happened?

just on many updates this issue appears:
* Error while installing package: tentative de remplacement de « /usr/share/metainfo/granite.appdata.xml », qui appartient aussi au paquet libgranite-common 6.2.0+r2223+pkg122~ubuntu6.1
* while trying just to update the system !
* eg to re-install code: 
* Error while installing package: tentative de remplacement de « /usr/share/metainfo/granite.appdata.xml », qui appartient aussi au paquet libgranite-common 6.2.0+r2223+pkg122~ubuntu6.1
* oops exactly the same.. i've tried to fix or re-install:
* sudo apt-get -y install libgranite5
Lecture des listes de paquets... Fait
Construction de l'arbre des dépendances       
Lecture des informations d'état... Fait
Les NOUVEAUX paquets suivants seront installés :
  libgranite5
0 mis à jour, 1 nouvellement installés, 0 à enlever et 0 non mis à jour.
120 partiellement installés ou enlevés.
Il est nécessaire de prendre 0 o/167 ko dans les archives.
Après cette opération, 650 ko d'espace disque supplémentaires seront utilisés.
(Lecture de la base de données... 121290 fichiers et répertoires déjà installés.)
Préparation du dépaquetage de .../libgranite5_5.3.0-1_amd64.deb ...
Dépaquetage de libgranite5:amd64 (5.3.0-1) ...
dpkg: erreur de traitement de l'archive /var/cache/apt/archives/libgranite5_5.3.0-1_amd64.deb (--unpack) :
 tentative de remplacement de « /usr/share/metainfo/granite.appdata.xml », qui appartient aussi au paquet libgranite-common 6.2.0+r2223+pkg122~ubuntu6.1
Des erreurs ont été rencontrées pendant l'exécution :
 /var/cache/apt/archives/libgranite5_5.3.0-1_amd64.deb
* also done this :  --fix-broken install and nothing repaired

### Steps to Reproduce

just try to install the update or use the updater app
the message about libgranite appears: nothing else !

for the clue is the fact i would like to install pantheon-tweaks .. which permits the little reduce button for the windows - really i HATE apple tricks ! nothing but is clean for sweepers ! nothing to get inside better - so the keys i've not.

### Expected Behavior

just update easy

### OS Version

6.x (Odin)

### Software Version

Latest release (I have run all updates)

### Log Output

```shell
Error while installing package: tentative de remplacement de « /usr/share/metainfo/granite.appdata.xml », qui appartient aussi au paquet libgranite-common 6.2.0+r2223+pkg122~ubuntu6.1
```

### Hardware Info

sure not the problem !
![image](https://user-images.githubusercontent.com/449844/187029785-431cf95c-41ba-4ada-b122-f90f3ad69081.png)

**Comment by peteruithoven** [comment]
This is a bit hard to follow, not code highlighting (I've added some as an example) and it's all in French? 

> for the clue is the fact i would like to install pantheon-tweaks

But you haven't yet installed it? I don't see the relevance.

**Comment by peteruithoven** [comment]
Any change you've added a ppa that might be causing issues?

**Comment by peteruithoven** [comment]
Some relevant stack exchange questions:
- https://elementaryos.stackexchange.com/questions/29510/i-cant-install-any-update-this-error-comes-in-my-elementary-os
- https://elementaryos.stackexchange.com/questions/28521/why-cant-i-upgrade-libgranite5-in-the-appcenter
- https://elementaryos.stackexchange.com/questions/29110/trying-to-install-tweaks-on-odin-6-install-error-writing-libgranite5
- https://elementaryos.stackexchange.com/questions/29226/dependency-problem-with-libgranite

**Comment by AkoZ** [comment]
Y thank yu for trying to help ! .. the only big issue is that is makes elementary OS no update all.
The principal thing is :
Error while installing package: try to replace « /usr/share/metainfo/granite.appdata.xml », which is package from libgranite-common 6.2.0+r2223+pkg122~ubuntu6.1
But i'll read and try the solutions there ... may be then update this.

PS: oops: may be i've added a PPA that might be causing issues, but how to find the BAD PPA and then remove it ? @peteruithoven excuse, but if you can find the tricker ?

`~$ grep ^ /etc/apt/sources.list /etc/apt/sources.list.d/*
/etc/apt/sources.list:# deb cdrom:[elementary OS 6.1 _odin_ - stable amd64 (20211218)]/ focal contrib main non-free
/etc/apt/sources.list:deb mirror://mirrors.ubuntu.com/mirrors.txt focal main restricted universe multiverse
/etc/apt/sources.list:deb-src mirror://mirrors.ubuntu.com/mirrors.txt focal main restricted universe multiverse
/etc/apt/sources.list:deb mirror://mirrors.ubuntu.com/mirrors.txt focal-updates main restricted universe multiverse
/etc/apt/sources.list:deb-src mirror://mirrors.ubuntu.com/mirrors.txt focal-updates main restricted universe multiverse
/etc/apt/sources.list:
/etc/apt/sources.list.d/amdgpu.list:deb https://repo.radeon.com/amdgpu/22.20/ubuntu focal main
/etc/apt/sources.list.d/amdgpu.list:# deb-src https://repo.radeon.com/amdgpu/22.20/ubuntu focal main
/etc/apt/sources.list.d/amdgpu.list.save:deb https://repo.radeon.com/amdgpu/22.20/ubuntu focal main
/etc/apt/sources.list.d/amdgpu.list.save:# deb-src https://repo.radeon.com/amdgpu/22.20/ubuntu focal main
/etc/apt/sources.list.d/amdgpu-proprietary.list:# Enabling this repository requires acceptance of the following license:
/etc/apt/sources.list.d/amdgpu-proprietary.list:# /usr/share/amdgpu-install/AMDGPUPROEULA
/etc/apt/sources.list.d/amdgpu-proprietary.list:# deb https://repo.radeon.com/amdgpu/22.20/ubuntu focal proprietary
/etc/apt/sources.list.d/amdgpu-proprietary.list.save:# Enabling this repository requires acceptance of the following license:
/etc/apt/sources.list.d/amdgpu-proprietary.list.save:# /usr/share/amdgpu-install/AMDGPUPROEULA
/etc/apt/sources.list.d/amdgpu-proprietary.list.save:# deb https://repo.radeon.com/amdgpu/22.20/ubuntu focal proprietary
/etc/apt/sources.list.d/appcenter.list:deb http://packages.elementary.io/appcenter jolnir main
/etc/apt/sources.list.d/appcenter.list.save:deb http://packages.elementary.io/appcenter jolnir main
/etc/apt/sources.list.d/elementary.list:deb http://ppa.launchpad.net/elementary-os/stable/ubuntu focal main
/etc/apt/sources.list.d/elementary.list:deb-src http://ppa.launchpad.net/elementary-os/stable/ubuntu focal main
/etc/apt/sources.list.d/elementary.list:
/etc/apt/sources.list.d/elementary.list.save:deb http://ppa.launchpad.net/elementary-os/stable/ubuntu focal main
/etc/apt/sources.list.d/elementary.list.save:deb-src http://ppa.launchpad.net/elementary-os/stable/ubuntu focal main
/etc/apt/sources.list.d/elementary.list.save:
/etc/apt/sources.list.d/elementary-os-ubuntu-stable-focal.list:# deb-src http://ppa.launchpad.net/elementary-os/stable/ubuntu focal main
/etc/apt/sources.list.d/elementary-os-ubuntu-stable-focal.list.save:# deb-src http://ppa.launchpad.net/elementary-os/stable/ubuntu focal main
/etc/apt/sources.list.d/patches.list:deb http://ppa.launchpad.net/elementary-os/os-patches/ubuntu focal main
/etc/apt/sources.list.d/patches.list:deb-src http://ppa.launchpad.net/elementary-os/os-patches/ubuntu focal main
/etc/apt/sources.list.d/patches.list:
/etc/apt/sources.list.d/patches.list.save:deb http://ppa.launchpad.net/elementary-os/os-patches/ubuntu focal main
/etc/apt/sources.list.d/patches.list.save:deb-src http://ppa.launchpad.net/elementary-os/os-patches/ubuntu focal main
/etc/apt/sources.list.d/patches.list.save:
/etc/apt/sources.list.d/philip_scott-ubuntu-pantheon-tweaks-focal.list:deb http://ppa.launchpad.net/philip.scott/pantheon-tweaks/ubuntu focal main
/etc/apt/sources.list.d/philip_scott-ubuntu-pantheon-tweaks-focal.list:# deb-src http://ppa.launchpad.net/philip.scott/pantheon-tweaks/ubuntu focal main
/etc/apt/sources.list.d/rocm.list:deb [arch=amd64] https://repo.radeon.com/rocm/apt/5.2 ubuntu main
/etc/apt/sources.list.d/rocm.list.save:deb [arch=amd64] https://repo.radeon.com/rocm/apt/5.2 ubuntu main
/etc/apt/sources.list.d/ximion-ubuntu-packagekit-focal.list:deb http://ppa.launchpad.net/ximion/packagekit/ubuntu focal main
/etc/apt/sources.list.d/ximion-ubuntu-packagekit-focal.list:# deb-src http://ppa.launchpad.net/ximion/packagekit/ubuntu focal main
/etc/apt/sources.list.d/ximion-ubuntu-packagekit-focal.list.save:deb http://ppa.launchpad.net/ximion/packagekit/ubuntu focal main
/etc/apt/sources.list.d/ximion-ubuntu-packagekit-focal.list.save:# deb-src http://ppa.launchpad.net/ximion/packagekit/ubuntu focal main`

**Comment by peteruithoven** [comment]
PPA's are tricky business... This is why Flatpaks, snaps, AppImages etc are so great. 

First remove the thing you've installed through the PPA and them remove the PPA using `add-apt-repository -r`. 

So if you've installed pantheon-tweaks you'd:
```
sudo apt remove pantheon-tweaks
sudo add-apt-repository -r ppa:philip.scott/pantheon-tweaks
```
